src/Sylius/Component/Resource/Metadata/Registry.php

```diff
@@ -34,7 +39,7 @@ public function getAll()
      */
     public function get($alias)
     {
-        if (!array_key_exists($alias, $this->metadata)) {
+        if (!isset($this->metadata[$alias])) {
```

Comment:
True, but its (admitedly insignificant) overhead. Don't see any advantage in using the pulic API when both pathways are tested. But don't feel strongly about it.